model/otlpgrpc/logs_test.go

```diff
@@ -63,20 +63,93 @@ var logsRequestJSON = []byte(`
 	  ]
 	}`)
 
+var logsTransitionData = [][]byte{
+	[]byte(`
+	{
+	  "resourceLogs": [
+		{
+          "resource": {},
+		  "instrumentationLibraryLogs": [
+			{
+              "instrumentationLibrary": {},
+			  "logRecords": [
```

Comment:
There's a mix of tabs and spaces here that makes the JSON a bit hard to read. Likely inherited from the var above this one.

Not a blocker!

Reply:
Fixed